src/yang/example-compa-diffserv.yang

```diff
@@ -1,430 +1,431 @@
-   module example-compa-diffserv {
-     yang-version 1.1;
-     namespace "urn:ietf:params:xml:ns:yang:example-compa-diffserv";
-     prefix example;
+module example-compa-diffserv {
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:example-compa-diffserv";
+  prefix example;
 
-     import ietf-qos-classifier {
-       prefix classifier;
-       reference "RFC XXXX: YANG Model for QoS";
-     }
-     import ietf-qos-policy {
-       prefix policy;
-       reference "RFC XXXX: YANG Model for QoS";
-     }
-     import ietf-qos-action {
-       prefix action;
-       reference "RFC XXXX: YANG Model for QoS";
-     }
-     import ietf-diffserv {
-       prefix diffserv;
-       reference "RFC XXXX: YANG Model for QoS";
-     }
+  import iana-qos-types {
+    prefix qos-types;
+    reference
+      "RFC XXXX: YANG Models for Quality of Service (QoS).";
+  }
+  import ietf-traffic-policy {
+    prefix policy;
+    reference "RFC XXXX: YANG Model for QoS";
+  }
+  import ietf-qos-action {
+    prefix action;
+    reference "RFC XXXX: YANG Model for QoS";
+  }
+  import ietf-diffserv {
+    prefix diffserv;
+    reference "RFC XXXX: YANG Model for QoS";
+  }
 
-     organization "Company A";
-     contact
-       "Editor:   XYZ
-                 <mailto:xyz@compa.com>";
-     description
-       "This module contains a collection of YANG definitions of
-        companyA diffserv specification extension.";
-     revision 2016-03-03 {
-       description
-         "Initial revision for diffserv actions on network packets";
-       reference
-         "RFC 6020: YANG - A Data Modeling Language for the
-                    Network Configuration Protocol (NETCONF)";
-     }
+  organization "Company A";
+  contact
+    "Editor:   XYZ
+              <mailto:xyz@compa.com>";
+  description
+    "This module contains a collection of YANG definitions of
+     companyA diffserv specification extension.";
+  revision 2016-03-03 {

```

Comment:
s/2016-03-03/YYYY-MM-DD/ and the sed script in draft/Makefile can replace it with the current date.